run-app.sh

```diff
@@ -1,3 +1,13 @@
 #!/bin/sh
 
-java -Xmx${HEAPSPACE_MAX:-"1000m"} --add-exports java.desktop/sun.font=ALL-UNNAMED -jar /app.jar
\ No newline at end of file
+# Export docker profile to enable json logging when running in docker
+if [[ "$RUNNING_IN_DOCKER" = "true" ]]; then
+  PROVIDED_PROFILES=${SPRING_PROFILES_ACTIVE}
+  if [[ -z "$PROVIDED_PROFILES" ]]; then
+    export SPRING_PROFILES_ACTIVE="docker"
+  else
+    export SPRING_PROFILES_ACTIVE="docker,$PROVIDED_PROFILES"
+  fi
+fi
```

Comment:
Syns denne løsningen er litt teit, men det funker :shrug: 
Støtter `SPRING_PROFILES_ACTIVE` trailing komma så kunne den vært bittelitt penere :upside_down_face: 
```suggestion
if [[ "$RUNNING_IN_DOCKER" = "true" ]]; then
  export SPRING_PROFILES_ACTIVE="docker,$SPRING_PROFILES_ACTIVE"
fi
```